### ISSUE TITLE
Compatibility with latest versions of python

### ISSUE THREAD

**Issue by umarIft** [x-tabdeveloping/topicwizard#18]
Hi,
I am unable to install topic-wizard and I am using python 3.11. Is this a known issue? It gives an error at wordcloud package during the installation of the topic-wizard package.

**Comment by x-tabdeveloping** [comment]
Yes it's a known issue. I will try to fix it as soon as possible, in the meantime I suggest you  try to work in a virtual environment with 3.10.

**Comment by umarIft** [comment]
Thanks for the quick reply. I switched to python 3.10 and it worked :)

**Comment by x-tabdeveloping** [comment]
Fixed with version 0.5.0 :)
